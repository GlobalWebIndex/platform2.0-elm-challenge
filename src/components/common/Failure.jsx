import React from "react";

const Failure = ({ error }) => {
  if (error) {
    // Log to custom service maybe
    console.error("Ooops -- An error occured!", error);
  }

  return (
    <div>
      Oops... something must have gone really wrong!
      <br />
      What could have happened? Hope all the 🐈 are still fine!
    </div>
  );
};

export default Failure;
