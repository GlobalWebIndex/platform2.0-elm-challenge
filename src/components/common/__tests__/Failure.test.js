import { render } from "@testing-library/react";
import Failure from "../Failure";

describe("<Failure/>", () => {
  it("renders", () => {
    const { container } = render(<Failure />);
    expect(container).toMatchSnapshot();
  });
});
