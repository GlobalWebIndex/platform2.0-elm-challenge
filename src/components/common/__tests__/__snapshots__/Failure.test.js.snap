// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Failure/> renders 1`] = `
<div>
  <div>
    Oops... something must have gone really wrong!
    <br />
    What could have happened? Hope all the 🐈 are still fine!
  </div>
</div>
`;
